packages/core/node/core-node-server-internal/src/node_service.ts

```diff
@@ -33,6 +33,13 @@ export interface InternalNodeServicePreboot {
   roles: NodeRoles;
 }
 
+export interface InternalNodeServiceStart {
+  /**
+   * Retrieve the Kibana instance uuid.
+   */
```

Comment:
NIT: I know this comment was also on `InternalNodeServicePreboot`, but not sure what it is supposed to mean exactly 😅 ?

Reply:
copy-pasta 😅